app/push-sw.js

```diff
@@ -41,10 +40,30 @@ self.addEventListener('notificationclick', (evt) => {
   );
 });
 
+function recipientsToString(recipients) {
+  let result;
+  if (!recipients.length) {
+    result = 'Reminder';
```

Comment:
It's not, but you know what we say: Be conservative in what you do, be liberal in what you accept from others :)